### PR TITLE
Remove KUBECF_CHECKOUT dependency in make tests-kubecf

### PR DESCRIPTION
**Pull request by viccuad**
Use the same approach of SITS for the rest of tests in `make tests-kubecf`.

Tested by running smokes against a CaaSP4 cluster.

### CODE REVIEW

**Comment by viccuad** [comment]
For cats I get `quarksjobs.quarks.cloudfoundry.org "susecf-scf-cf-acceptance-tests" not found` so maybe I'm doing something deeply wrong.

**Comment by viccuad** [comment]
Will need an update of https://github.com/SUSE/catapult/wiki/KubeCF-testing.

**Comment by mudler** [comment]
> For cats I get `quarksjobs.quarks.cloudfoundry.org "susecf-scf-cf-acceptance-tests" not found` so maybe I'm doing something deeply wrong.

```
../kubectl/kubectl patch qjob susecf-scf-acceptance-tests --namespace scf --type merge --patch 'spec:

  trigger:

    strategy: now'
```
I think there is a typo indeed, should be `susecf-scf-acceptance-tests` not `susecf-scf-cf-acceptance-tests`

File: modules/tests/kubecf-test.sh [hunk 1]
```diff
@@ -31,35 +31,34 @@ wait_for_tests_pod() {
...
+    kubectl patch qjob "${KUBECF_DEPLOYMENT_NAME}"-sync-integration-tests --namespace "${KUBECF_NAMESPACE}" --type merge --patch 'spec:
+      trigger:
+          strategy: now'
+    info "Waiting for the sync-integration-tests pod to start..."
+    until sync_integration_tests_pod_name || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
+    if [[ "${timeout}" == 0 ]]; then return 1; fi
+    pod_name="$(sync_integration_tests_pod_name)"
+    container_name="sync-integration-tests-sync-integration-tests"
+else
+    kubectl patch qjob "${KUBECF_DEPLOYMENT_NAME}"-cf-acceptance-tests --namespace "${KUBECF_NAMESPACE}" --type merge --patch 'spec:
```
Comment (mudler) [inline modules/tests/kubecf-test.sh]:
```suggestion
    kubectl patch qjob "${KUBECF_DEPLOYMENT_NAME}"-acceptance-tests --namespace "${KUBECF_NAMESPACE}" --type merge --patch 'spec:
```

**Comment by viccuad** [comment]
thanks for the fast review!

**Pull request merged**